skimage/filters/thresholding.py

```diff
@@ -204,7 +206,8 @@ def threshold_yen(image, nbins=256):
     P1_sq = np.cumsum(pmf ** 2)
     # Get cumsum calculated from end of squared array:
     P2_sq = np.cumsum(pmf[::-1] ** 2)[::-1]
-    # P2_sq indexes is shifted +1. I assume, with P1[:-1] it's help avoid '-inf'
+    # P2_sq indexes is shifted +1.
+    # I assume, with P1[:-1] it's help avoid '-inf'
```

Comment:
I just split the line to respect PE8, flakes was complaining, but I should do better and fix the english as well.
It looks like that part escaped from a previous review.